documentation/modules/virt-migration-workflow.adoc

```diff
@@ -30,10 +32,30 @@ The workflow describes the following steps:
 +
 *After the VM disks are transferred:*
 
-. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it.
+. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it when importing from VMWare.
 +
 The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the target VM.
++
+. The `Migration Controller` service creates a `VirtualMachine` CR for each source virtual machine (VM), connected to the PVCs.
+
+. When the target VM is powered on, the `KubeVirt Controller` service creates a `virt-launcher` pod and a `VirtualMachineInstance` CR.
++
+The `virt-launcher` pod runs `QEMU-KVM` with the PVCs attached as VM disks.
 
+*Cold migration:*
+
+. When you create a `Migration` custom resource (CR) to run a migration plan, the `Migration Controller` service creates a `PersistentVolumeClaim` CR and a `OvirtVolumePopulator`/`OpenstackVolumePopulator` CR for each source VM disk.
++
+*For each VM disk:*
+
+. The `Populator Controller` service creates a temporarily persistent volume claim (PVC).
+. If the `StorageClass` has a dynamic provisioner, the persistent volume (PV) is dynamically provisioned by the `StorageClass` provisioner.
+. The `Migration Controller` service creates a dummy pod to bound the PVC.
```

Comment:
we can identify it by its name, right? let's write that
and "to bound" -> "to bind"